pkg/operator/resources/operator/operator_test.go

```diff
@@ -17,75 +19,80 @@ var crdTypeMap = map[string]interface{}{
 	"operator-crd": &cdiv1alpha1.CDI{},
 }
 
-func TestCRDSchemas(t *testing.T) {
-	for crdFileName, cdiObjType := range crdTypeMap {
+func TestOperatorResource(t *testing.T) {
```

Comment:
In general we try to have a single 'suite' file that has the 'Testing.T' in it. We name it: _suite_test.go with the package here being apiserver. It also allows us to set things like logger in the before suite. A decent example is in pkg/controller/controller_suite_test.go